CIPs/0009.md

```diff
@@ -0,0 +1,90 @@
+# CIP [0009]: Standalone signing service for Validators
+
+- Date: 2020-05-14
+- Author: @asaj
```

Comment:
Hi Asa @asaj thanks for submitting this proposal. Can you update your PR to reflect the new CIP template that's been activated as part of the new CIP process. Link to it here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md